### PR TITLE
Summary page

### PR DESCRIPTION
**Pull request by kndshein**

### CODE REVIEW

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Unhappiness ensues. 

Screen Shot 2021-08-04 at 8 39 19 PM

File: src/pages/summary.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Link } from "gatsby";
+
+export default function Summary({ location }) {
+  console.log(location);
```
[the remark this answers is not in the document]
Comment (kndshein) [reply src/pages/summary.jsx]: Done.

File: src/pages/summary.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <>
+      {location.state.corrects.map((word, index) => {
+        return (
+          <div key={index}>
+            {word}{" "}
+            {location.state.errors[word]?.map((error, index2) => {
+              return <div key={index2}>{error}</div>;
```
[the remark this answers is not in the document]
Comment (kndshein) [reply src/pages/summary.jsx]: Done.

**Review by daleinen7 — APPROVED** [review]
First coffee review from phone.

**Pull request merged**